tables/TaQL/TaQLNodeDer.cc

```diff
@@ -107,8 +107,6 @@ TaQLConstNodeRep::TaQLConstNodeRep (Int64 value, const String& subTableName)
     itsCValue      (value, 0.),
     itsSValue      (subTableName)
 {}
-TaQLConstNodeRep::~TaQLConstNodeRep()
```

Comment:
Next time please make a separate PR for cleaning up existing code. When this PR introduces a bug it's a lot less code to review to determine the bug.
I much rather review 2 PRs, one cleanup and one code fixes.

Reply:
Undid superfluous changes (I assume NFC means non-functional changes).

Reply:
NFC indeed is non-functional change.